sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -1520,33 +1542,33 @@ export interface ThreadMessageOutput {
 
 // @public
 export interface ThreadRunOutput {
-    assistant_id: string;
-    cancelled_at: number | null;
-    completed_at: number | null;
-    created_at: number;
-    expires_at: number | null;
-    failed_at: number | null;
+    assistantId: string;
+    cancelledAt: Date | null;
```

Comment:
These `null` types are still bothering me but if that's what the TypeSpec defines then the SDK should stay true to it. I am surprised that the TypeSpec is defined as such, but I am not going to block PRs on that 👍

Reply:
Yeah, I was curious if the service was actually sending literal `null` values in the JSON. Maybe @deyaaeldeen knows from his OAI experience?

Reply:
@xirzec / @maorleger  The typespec is based on the open ai assistants which has them defined as null.  I linked the spec here
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/ai/OpenAI.Assistants/assistants/models.tsp